projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/page-meta.service.migration.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  GET_META,
+  GET_META_RESOLVER,
+  PAGE_META_SERVICE,
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+// projects/core/src/cms/facade/page-meta.service.ts
+export const PAGE_META_SERVICE_MIGRATION: MethodPropertyDeprecation[] = [
```

Comment:
I think `PAGE_META_SERVICE_MIGRATION` needs to be registered somewhere for this to work.